yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -281,7 +282,40 @@ def run_ffmpeg_multiple_files(self, input_paths, out_path, opts, **kwargs):
             [(path, []) for path in input_paths],
             [(out_path, opts)], **kwargs)
 
-    def real_run_ffmpeg(self, input_path_opts, output_path_opts, *, expected_retcodes=(0,)):
+    def parse_ffmpeg_time_string(time_string):
+        time = 0
+        reg1 = re.match(r"((?P<H>\d\d?):)?((?P<M>\d\d?):)?(?P<S>\d\d?)(\.(?P<f>\d{1,3}))?", time_string)
+        reg2 = re.match(r"\d+(?P<U>s|ms|us)", time_string)
+        if reg1:
+            if reg1.group('H') is not None:
+                time += 3600 * int(reg1.group('H'))
+            if reg1.group('M') is not None:
+                time += 60 * int(reg1.group('M'))
+            time += int(reg1.group('S'))
+            if reg1.group('f') is not None:
+                time += int(reg1.group('f')) / 1_000
+        elif reg2:
+            time = int(reg2.group('U'))
+            if reg2.group('U') == 'ms':
+                time /= 1_000
+            elif reg2.group('U') == 'us':
+                time /= 1_000_000
+        return time
+
+    def compute_prefix(match):
+        res = int(match.group('E'))
+        if match.group('f') is not None:
+            res += int(match.group('f'))
+        if match.group('U') is not None:
+            if match.group('U') == 'g':
+                res *= 1_000_000_000
+            elif match.group('U') == 'm':
+                res *= 1_000_000
+            elif match.group('U') == 'k':
+                res *= 1_000
+        return res
```

Comment:
Repeat of the functions already present in yt_dlp/downloader/external.py.
Don't repeat yourself otherwise the day we'll have to make some changes it will have to be repeated within both files even though those are the exact same functions.